internal/db/schema/migrations/oss/postgres/28/01_prior_session_trigger.up.sql

```diff
@@ -0,0 +1,113 @@
+begin;
```

Comment:
you are going to want to rename this file as there already is a `28/01` migration in main.